language/predefined/closure/bindto.xml

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- $Revision$ -->
+<!-- EN-Revision: 9c74079f12d67cabb52c124d761f48275417d7eb Maintainer: samesch Status: ready -->
+<!-- Reviewed: no -->
+<refentry xml:id="closure.bindto" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
+ <refnamediv>
+  <refname>Closure::bindTo</refname>
+  <refpurpose>
+   Dupliziert eine Closure mit einem neuen gebundenen Objekt und Geltungsbereich einer Klasse
+  </refpurpose>
+ </refnamediv>
+
+ <refsect1 role="description">
+  &reftitle.description;
+  <methodsynopsis role="Closure">
+   <modifier>public</modifier> <type class="union"><type>Closure</type><type>null</type></type><methodname>Closure::bindTo</methodname>
+   <methodparam><type class="union"><type>object</type><type>null</type></type><parameter>newThis</parameter></methodparam>
+   <methodparam choice="opt"><type class="union"><type>object</type><type>string</type><type>null</type></type><parameter>newScope</parameter><initializer>"static"</initializer></methodparam>
+  </methodsynopsis>
+  <para>
+   Erzeugt eine neue
+   <link linkend="functions.anonymous">anonyme Funktion</link> mit demselben
+   Funktionskörper und denselben gebundenen Variablen wie diese und gibt sie
+   zurück, aber möglicherweise mit einem anderen gebundenen Objekt und einem
+   neuen Geltungsbereich der Klasse.
```

Comment:
Somehow that is very awkwardly expressed. 

`diese` has no real reference to anything for me. But presumably the English translation has to be refined first so that this can be adapted (it is also misleading for me).

Reply:
For me this refers to the closure that's duplicated. But you're right.